internal/pkg/aws/elbv2/elbv2.go

```diff
@@ -73,6 +74,27 @@ func (e *ELBV2) ListenerRuleHostHeaders(ruleARN string) ([]string, error) {
 	return hostHeaders, nil
 }
 
+// ListenerRuleIsRedirect returns true if ruleARN has an action to redirect.
+func (e *ELBV2) ListenerRuleIsRedirect(ctx context.Context, ruleARN string) (bool, error) {
+	resp, err := e.client.DescribeRules(&elbv2.DescribeRulesInput{
+		RuleArns: aws.StringSlice([]string{ruleARN}),
+	})
+	switch {
+	case err != nil:
+		return false, fmt.Errorf("get listener rule for %s: %w", ruleARN, err)
+	case len(resp.Rules) == 0:
+		return false, fmt.Errorf("cannot find listener rule %s", ruleARN)
```

Comment:
nit:
```suggestion
		return false, fmt.Errorf("listener rule %q not found", ruleARN)
```

Reply:
changed the error when changing the API https://github.com/aws/copilot-cli/pull/4017#discussion_r976787597